camel/societies/workforce/workforce_logger.py

```diff
@@ -50,6 +63,27 @@ def __init__(self, workforce_id: str):
         self._worker_information: Dict[str, Dict[str, Any]] = {}
         self._initial_worker_logs: List[Dict[str, Any]] = []
 
+    def _get_color_message(self, event: LogEvent) -> str:
+        r"""Gets a colored message for a log event."""
+        if event.color is None or event.color not in _COLOR_MAP:
+            return event.message
+        color = _COLOR_MAP.get(event.color)
+        return f"{color}{event.message}{Fore.RESET}"
+
+    def log_message(self, event: LogEvent) -> None:
```

Comment:
For the default WorkforceLogger, we should keep using print to ensure refactoring doesn’t change the output behavior.